docs/REST API/rest-api.md

```diff
@@ -4160,22 +4160,22 @@ Get aggregate status counts of MDM profiles applying to macOS hosts enrolled to
 
 ### Run custom MDM command
 
-This endpoint tells Fleet to run a custom MDM command, on the targeted macOS hosts, the next time they come online.
+This endpoint tells Fleet to run a custom MDM command, on the targeted macOS or Windows hosts, the next time they come online.
 
-`POST /api/v1/fleet/mdm/apple/enqueue`
+`POST /api/v1/fleet/mdm/commands`
```

Comment:
I like `enqueue`. It's more explicit. Tells the user that they're adding a command to the queue (doesn't necessarily run right away because the host might be offline)

Reply:
The way I was thinking about this was to simplify so you have the same URL for POST and GET operations. Having `/enqueue` works for me as well.

Reply:
> same URL for POST and GET operations.

Ah, I see. That's smart. 

Although this endpoint isn't a "normal" CRUD because it doesn't just create a command, it queues/runs one.

I still like `enqueue` a bit more because it's explicit.

What do you think?